README.md

```diff
@@ -163,6 +163,10 @@ GoTTY uses [hterm](https://groups.google.com/a/chromium.org/forum/#!forum/chromi
 
 ## Alternatives
 
+### Command line client
+
+* [gotty-client](https://github.com/moul/gotty-client): If you want to connect to an official GoTTY server from your terminal
```

Comment:
Could you remove `an official`? Because this repository is official and this word might make readers confused.